x-pack/plugins/security_solution/public/timelines/components/graph_overlay/index.tsx

```diff
@@ -29,20 +29,28 @@ import { SourcererScopeName } from '../../../common/store/sourcerer/model';
 import { useSourcererDataView } from '../../../common/containers/sourcerer';
 import { sourcererSelectors } from '../../../common/store';
 
-const OverlayContainer = styled.div`
+const OverlayStyle = `
   display: flex;
   flex-direction: column;
   flex: 1;
   width: 100%;
 `;
 
-const FullScreenOverlayContainer = styled.div`
+const OverlayContainer = styled.div`
+  ${OverlayStyle}
+`;
+
+const FullScreenOverlayStyles = `
   position: fixed;
   top: 0;
   bottom: 0;
   left: 0;
   right: 0;
-  z-index: ${(props) => props.theme.eui.euiZLevel3};
+  z-index: 3000;
```

Comment:
wasn't sure how to reach `theme.eui.euiZLevel3` here 🤔

Reply:
I wanted to reuse the styles in the `useLayoutEffect` below to update the style of the `OverlayContainer` to full screen.

I did

```
const FullScreenOverlayStyles = css`
  position: fixed;
  top: 0;
  bottom: 0;
  left: 0;
  right: 0;
  z-index: ${(props) => props.theme.eui.euiZLevel3};
`;
...
// useLayoutEffect
sessionContainerRef.current.setAttribute('style', FullScreenOverlayStyles.join(''));
```
but it seems like `FullScreenOverlayStyles` doesn't get evaluated at the time
<img width="515" alt="image" src="https://user-images.githubusercontent.com/16872649/162263629-46d84857-5d5e-4c14-9782-09aff1dcb364.png">

Reply:
ah lame, oh well. I guess theres no other way to get at euiTheme vars? outside of styled-components?

Reply:
e.g.

you could call   const { euiTheme } = useEuiTheme(); and then define your fullscreen css at runtime instead of ouside as a const.

Reply:
you can try to import it from `@kbn/ui-theme`

Reply:
I tried the `useTheme` hook which is supposed to be the official way to use theming outside of styled-component, but unfortunately it doesn't seem to like typescript :(

Reply:
@mitodrummer I tried the `useTheme` hook which is supposed to be the official way to use theming outside of styled-component, but unfortunately it doesn't seem to like typescript 😞 

@patrykkopycinski looks like thats exactly what i need haha, thanks!

Reply:
> e.g.
> 
> you could call const { euiTheme } = useEuiTheme(); and then define your fullscreen css at runtime instead of ouside as a const.

I agree, that would normally be a good idea. in this case the styled component `<FullScreenOverlayContainer />` consuming the `FullScreenOverlayStyles` is also being used below elsewhere, so it might not be a bad idea to keep it outside for this case